### ISSUE TITLE
Improve ONNX support with BatchNormalization

### ISSUE THREAD

**Issue by francoisWeber** [vespa-engine/vespa#14845]
Hi Vespa Team, 
I'm using PyTorch to implement learning-to-rank for Vespa. I use ONNX models as serialization medium between PyTorch and Vespa.
However, it seems that a really useful ONNX operation is not yet supported inside Vespa : [BatchNormalization](https://github.com/onnx/onnx/blob/master/docs/Operators.md#BatchNormalization). It would be really usefull to add support for this operation soon 🙂

Best regards.
François Weber

**Comment by jobergum** [comment]
Thanks a lot for your interest in Vespa.ai, 

We recently integrated ONNX-RT in Vespa. The documentation is not up to date with this change yet but we are fixing that this sprint. You can see an example here https://github.com/vespa-engine/sample-apps/blob/master/dense-passage-retrieval-with-ann/src/main/application/schemas/wiki.sd#L48-L55

Using **onnxModel()** will use the onnx-rt, while **onnx()** uses the Vespa implementation.

**Comment by francoisWeber** [comment]
Thanks !

**Comment by jobergum** [comment]
Thank you @francoisWeber, we will publish more on the onnx-rt integration and how to export PyTorch models to ONNX, how to optimize the ONNX graph (Graph optimizations, quantization etc) to improve run time serving performance. On CPU still, but see https://github.com/vespa-engine/vespa/issues/14406 for GPU. If you like us to implement GPU support feel free to add your cases to mentioned ticket. Thank you again.

**Comment by francoisWeber** [comment]
Hi Vespa team, 
Sorry for re-opening this issue but I still have some trouble using the ONNX-RT feature that @jobergum spoke about recently. With the Vespa image` vespaengine/vespa:7.343.1`, I'm using an ONNX network involving `BatchNormalization` layers within my search-definition:
```
    onnx-model onnx_model {
        file: models/nn_bipedal_cuckoo_9fa0_10feats_7a1e.onnx
        input "input_10feats_7a1e": attribute(adult)
        output "output": output
    }

    rank-profile onnx_profile {
        first-phase {
            expression: onnxModel(onnx_model).output
        }
    }
```
This raises the following exception when deploying the new search-definition:
```
java.lang.IllegalArgumentException: Could not import nn_bipedal_cuckoo_9fa0_10feats_7a1e: Operation 'BatchNormalization' is currently not implemented
at ai.vespa.rankingexpression.importer.onnx.GraphImporter.verifyNoWarnings(GraphImporter.java:272)
at ai.vespa.rankingexpression.importer.onnx.GraphImporter.importGraph(GraphImporter.java:155)
at ai.vespa.rankingexpression.importer.onnx.OnnxImporter.importModel(OnnxImporter.java:34)
at ai.vespa.rankingexpression.importer.ModelImporter.importModel(ModelImporter.java:45)
at ai.vespa.rankingexpression.importer.ModelImporter.importModel(ModelImporter.java:35)
at ai.vespa.rankingexpression.importer.configmodelview.ImportedMlModels.lambda$importRecursively$0(ImportedMlModels.java:69)
at java.base/java.util.stream.ForEachOps$ForEachOp$OfRef.accept(ForEachOps.java:183)
at java.base/java.util.stream.SortedOps$SizedRefSortingSink.end(SortedOps.java:357)
at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:485)
[...]
```
Do you have any clue to fix this problem and properly use ONNX-RT inference with BatchNorms ?
Thanks ! 
François Weber

**Comment by lesters** [comment]
@francoisWeber Put your model in a different directory than `models`, for instance in `files`, then this should work as expected. Models in the `models` directory triggers our old model import code, unfortunately this hasn't been fixed yet but will be soon.

**Comment by lesters** [comment]
@francoisWeber With the change in https://github.com/vespa-engine/vespa/pull/16176, this should no longer be a problem. Have you been able to test this yet?

**Comment by francoisWeber** [comment]
Thanks @lesters it works ! 🎉

**Comment by jobergum** [comment]
Great @francoisWeber, thanks for the feedback
